### PR TITLE
Update hostpath driver to latest tag in deployments

### PR DESCRIPTION
**Pull request by msau42**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
So that we can start testing new features added to hostpath in kubernetes-csi like topology, expansion, cloning

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by msau42** [comment]
Opening https://github.com/kubernetes-csi/csi-driver-host-path/issues/96 for test failure.

/retest

File: deploy/kubernetes-1.13/hostpath/csi-hostpath-plugin.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ spec:
             name: csi-data-dir
 
         - name: hostpath
-          image: quay.io/k8scsi/hostpathplugin:v1.1.0
```
Comment (davidz627) [inline deploy/kubernetes-1.13/hostpath/csi-hostpath-plugin.yaml]: do we not want to continue to "verify" `v1.1.0` compatibility with `k8s 1.13`? I guess what if a patch in `1.13` breaks compat?

Comment (msau42) [reply deploy/kubernetes-1.13/hostpath/csi-hostpath-plugin.yaml]: At least for hostpath driver, we have not been maintaining older releases.  Patches in K8s that break CSI are tested as part of K8s in-tree test configs, which will continue pulling older versions of the hostpath driver.

**Pull request merged**